sycl/plugins/unified_runtime/ur/adapters/hip/enqueue.cpp

```diff
@@ -82,6 +82,61 @@ void simpleGuessLocalWorkSize(size_t *ThreadsPerBlock,
     --ThreadsPerBlock[0];
   }
 }
+
+ur_result_t setHipMemAdvise(const void *DevPtr, size_t Size,
```

Comment:
does this want to be `static`?

Reply:
Good point! Actually, initially, I was going to make it static until I realised it's translation-unit-local in an anonymous namespace similarly to the other helper functions defined above this one.
I am open to suggestions and corrections as my understanding on this use the anonymous namespace may be incorrect.